include/FileManager.hpp

```diff
@@ -2,10 +2,9 @@
 #include <iostream>
 #include <string>
 #include <vector>
-#include <dirent.h>
-#include <cassert>
 #include <fstream>
-#include <cstring>
+#include<iterator>
+#include<boost/filesystem.hpp>
```

Comment:
We usually follow the format: `#include <iterator>` instead of `#include<iterator>` (notice the spaces) for readability.